scalatest/src/main/scala/org/scalatest/tools/DashboardReporter.scala

```diff
@@ -187,7 +187,7 @@ private[scalatest] class DashboardReporter(directory: String,
   //
   def purgeDir(directory: File, prefix: String): Unit = {
     directory.
-      listFiles().
+      listFiles().map(_.nn).
```

Comment:
**Java method returns array**
Java method returns array of type `Array[Null | T]`. We need to map it to `Array[T]`.